data/json/recipes/food/offal_dishes.json

```
@@ -198,7 +198,7 @@
     "difficulty": 4,
     "time": "25 m",
     "batch_time_factors": [ 83, 5 ],
-    "book_learn": [ [ "cookbook", 2 ] ],
+    "book_learn": [ [ "cookbook", 2 ], [ "scots_cookbook", 2 ] ],
```

```suggestion
    "book_learn": [ [ "cookbook", 2 ], [ "cookbook_liverforkids", 2 ] ],
```
I can't find much reference to scottish recipes for brain, is there something you're referring to here?

---

https://www.scotsman.com/news-2-15012/8-old-scottish-recipes-you-ve-probably-not-tried-1-3921086

It sounded like Brain Cakes were just fried brains and we don't have a tongue part yet.